compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/UIKitTextInputService.uikit.kt

```diff
@@ -274,32 +257,44 @@ internal class UIKitTextInputService(
         onCutRequested: (() -> Unit)?,
         onSelectAllRequested: (() -> Unit)?
     ) {
-        textUIView?.let {
-            val skiaRect = with(densityProvider()) {
-                org.jetbrains.skia.Rect.makeLTRB(
-                    l = rect.left / density,
-                    t = rect.top / density,
-                    r = rect.right / density,
-                    b = rect.bottom / density,
-                )
-            }
-            it.showTextMenu(
-                targetRect = skiaRect,
-                textActions = object : TextActions {
-                    override val copy: (() -> Unit)? = onCopyRequested
-                    override val cut: (() -> Unit)? = onCutRequested
-                    override val paste: (() -> Unit)? = onPasteRequested
-                    override val selectAll: (() -> Unit)? = onSelectAllRequested
-                }
+        val skiaRect = with(densityProvider()) {
+            org.jetbrains.skia.Rect.makeLTRB(
+                l = rect.left / density,
+                t = rect.top / density,
+                r = rect.right / density,
+                b = rect.bottom / density,
             )
         }
+
+        if (textUIView == null ) {
+            // If showMenu() is called and textUIView is not created,
+            // then it means that showMenu() called in SelectionContainer without any textfields,
+            // and IntermediateTextInputView must be created to show an editing menu
+            attachIntermediateTextInputView()
+            textUIView?.becomeFirstResponder()
+            updateView()
+        }
+
+        textUIView?.showTextMenu(
+            targetRect = skiaRect,
```

Comment:
Change type of this paramenter in `showTextMenu` to `CValue<CGRect>` and use
```suggestion
            targetRect = rect.toDpRect(densityProvider()).asCGRect(),
```

Reply:
applied your patch